Makefile

```diff
@@ -23,4 +25,14 @@ docs-%:
 
 include notebooks/subdir.mk
 
-.PHONY: all lint lint-non-init lint-init test mypy
+slow: fast run_notebooks docker
+	virtualenv -p python .venv
```

Comment:
virtualenv is not guaranteed to be installed.  also, `.venv` is where i stored my virtualenv.  can you make this:

```
python -m venv .notebooks-exec-env
.notebooks-exec-env/bin/pip install -r REQUIREMENTS-NOTEBOOK.txt
.notebooks-exec-env/bin/pip install starfish
make PYTHON=.notebooks-exec-env/bin/python run_notebooks
```